src/model/gateway.rs

```diff
@@ -689,15 +697,15 @@ impl GatewayIntents {
     pub const fn privileged() -> GatewayIntents {
         // bitflags don't support const evaluation. Workaround.
         // See: https://github.com/bitflags/bitflags/issues/180
-        Self::from_bits_truncate(Self::GUILD_MEMBERS.bits() | Self::GUILD_PRESENCES.bits())
+        Self::GUILD_MEMBERS.union(Self::GUILD_PRESENCES).union(Self::MESSAGE_CONTENT)
     }
 
     /// Checks if any of the included intents are privileged
     ///
     /// [GUILD_MEMBERS]: #associatedconstant.GUILD_MEMBERS
     /// [GUILD_PRESENCES]: #associatedconstant.GUILD_PRESENCES
     pub fn is_privileged(self) -> bool {
-        self.guild_members() || self.guild_presences()
+        self.guild_members() || self.guild_presences() || self.message_content()
     }
```

Comment:
`Self::privileged().contains(self)` may be better so we don't have to keep is_privileged in sync with privileged

Reply:
Included that in the latest commit along with some other changes that seemed too small to open a separate PR for